lib/Dist/Zilla/Plugin/UploadToCPAN.pm

```diff
@@ -221,6 +221,30 @@ has upload_uri => (
   predicate => 'has_upload_uri',
 );
 
+=attr retries
+
+The number of retries to perform on upload failure. This value will be passed to
+L<CPAN::Uploader>.
+
+=cut
+
+has retries => (
+  is => 'ro',
```

Comment:
I set it to 3 retries and 5 seconds delay. BTW, ugh, I pushed the repo on my laptop which contains some extra commits :-(